src/LaunchDarkly.JsonStream/ReaderAdapters.cs

```diff
@@ -117,7 +118,7 @@ private AnyValue FromNumber(double value, ValueType? typeHint)
                 switch (typeHint)
                 {
                     case ValueType.String:
-                        return AnyValue.String(value.ToString());
+                        return AnyValue.String(value.ToString(CultureInfo.InvariantCulture));
```

Comment:
Ran analysis to find this one as well. It has always been a warning to have ToString without a culture for something that can be a number.

This one probably matters less. Because it is for the case where we expect a string, but there was a number in the JSON. Which is already an edge case.